### DISCUSSION TITLE
Feature Request: Make RuleStats Text clickable.

### DISCUSSION THREAD

**Discussion by joeblubaugh** [grafana/grafana#48026] [Alerting]
When I'm looking at a list of rules, there's this nice colored text from RuleStats (see attached). In addition to the radio group, it'd be great to be able to click on some of this text to filter the list. My eye is already drawn there because I'm scanning for the overall state of my rules. I keep *trying* to click on those and then moving focus up and to the right to the radio button group

https://user-images.githubusercontent.com/1951900/164386740-4cb74cb7-c742-4b80-96a3-4f723ebeb90b.mov

.

**Comment by jessover9000** [comment]
Now that I look at the filters + Stats in combination, I almost feel like it’s an unnecessary duplication to both have the filters in the top but without item count AND the clickable StatsText. The state filters should only be available for Alert rules to begin with, and making the RuleStats clickable also opens up the question of “why can we not currently filter for errors?“. 

BUT you are right that these are all just theses and we haven’t tested with users how people actually use the filters and the list in general. It’s on my list for Q2 to run usability tests for the alert list and how people work with it, so maybe we can hold off on this until we have better insights on user pains and needs with this list. 

If we do have the resources, adding filter links to the stats text before reworking the whole thing is fine by me.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
